catalog/app/containers/NavBar/NavBar.tsx

```diff
@@ -389,13 +389,13 @@ function useLinks(): LinkDescriptor[] {
       label: 'URI',
       path: paths.uriResolver,
     },
-    { href: URLS.docs, label: 'Docs' },
+    cfg.mode !== 'PRODUCT' && { href: URLS.docs, label: 'Docs' },
```

Comment:
i think we should keep the documentation link

Reply:
Aneesh asked to remove it. I thought we should keep it too.

But after removing all marketing blocks docs link in the footer become visible now, and probably redundant

Reply:
Also, we have docs links in-place where something goes wrong and user should want to check the docs :)